content/blog/2021-07-12-500-contributors-mark/index.md

```diff
@@ -39,5 +39,101 @@ We nearly double the number of contributors in the last year, from ~300 to over
+|divyakumarjain|dmsolr|dominicqi|donbing007|dsc6636926|
+|dvsv2|dzx2018|echooymxq|efekaptan|elk-g|
+|eoeac|evanxuhe|feelwing1314|fgksgf|fredster33|
+|fuhuo|fulmicoton|geektcp|geomonlin|ggndnn|
+|gitter-badger|givingwu|glongzh|gnr163|gonedays|
+|grissom-grissom|grissomsh|guodongq|guyukou|gxthrj|
+|gy09535|gzshilu|hailin0|hanahmily|haotian2015|
+|haoyann|hardzhang|harvies|heihaozi|hepyu|
+|heyanlong|hi-sb|honganan|horber|hsoftxl|
+|huangyoje|huliangdream|huohuanhuan|innerpeacez|itsvse|
+|jasper-zsh|jbampton|jialong121|jinlongwang|jjlu521016|
+|jjtyro|jmjoy|jsbxyyx|justeene|juzhiyuan|
+|jy00464346|kaanid|kagaya85|karott|kayleyang|
+|kevinyyyy|kezhenxu94|kikupotter|kilingzhang|killGC|
+|kkl129|klboke|ksewen|kuaikuai|kun-song|
+|kylixs|landonzeng|langke93|langyan1022|langyizhao|
+|lazycathome|leemove|leizhiyuan|libinglong|lijial|
+|lilien1010|limfriend|linkinshi|linliaoy|liqiangz|
+|liu-junchi|liuhaoXD|liuhaoyang|liuyanggithup|liuzhengyang|
+|liweiv|lkxiaolou|llissery|louis-zhou|lpcy|
+|lpf32|lsyf|lucperkins|lujiajing1126|lunamagic1978|
+|lunchboxav|lxliuxuankb|lytscu|lyzhang1999|mage3k|
+|makefriend8|makingtime|mantuliu|maolie|margauxcabrera|
+|masterxxo|maxiaoguang64|membphis|mestarshine|mgsheng|
+|michaelsembwever|mikkeschiren|misaya|momo0313|moonming|
+|mrproliu|muyun12|nacx|neatlife|neeuq|
+|nic-chen|nickwongwong|nikitap492|nileblack|nisiyong|
+|novayoung|oatiz|oflebbe|olzhy|onecloud360|
+|osiriswd|panniyuyu|peng-yongsheng|pengweiqhca|potiuk|
+|probeyang|purgeyao|qijianbo010|qinhang3|qiuyu-d|
+|qjgszzx|qq362220083|qqeasonchen|qxo|ralphgj|
+|raybi-asus|refactor2|remicollet|rlenferink|rootsongjc|
+|rovast|ruibaby|s00373198|scolia|sdanzo|
+|seifeHu|sergicastro|shiluo34|sikelangya|simonlei|
+|sk163|snakorse|songzhendong|songzhian|sonxy|
+|spacewander|stalary|stenio2011|stevehu|stone-wlg|
+|sungitly|surechen|swartz-k|sxzaihua|tangxqa|
+|tanjunchen|tankilo|taskmgr|tbdpmi|terranhu|
+|terrymanu|tevahp|thanq|thebouv|tianyk|
+|tianyuak|tincopper|tinyu0|tom-pytel|tristaZero|
+|tristan-tsl|trustin|tsuilouis|tuohai666|tzsword-2020|
+|tzy1316106836|undefined|vcjmhg|viktoryi|vision-ken|
+|viswaramamoorthy|wallezhang|wang-yeliang|wangrzneu|wankai123|
+|wbpcode|web-xiaxia|webb2019|weiqiang-w|weiqiang333|
+|wendal|wengangJi|wenjianzhang|whfjam|whl12345|
+|willseeyou|wilsonwu|wind2008hxy|wingwong-knh|withlin|
+|wl4g|wqr2016|wu-sheng|wuguangkuo|wujun8|
+|wuwen5|wuxingye|x22x22|xbkaishui|xcaspar|
+|xdRight|xiaoxiangmoe|xiaoy00|xinfeingxia85|xingren23|
+|xinzhuxiansheng|xonze|xuanyu66|xuchangjunjx|xudianyang|
+|yanbw|yanfch|yang-xiaodong|yangxb2010000|yanickxia|
+|yanmaipian|yanmingbi|yantaowu|yaojingguo|yaowenqiang|
+|yazong|ychandu|ycoe|yimeng|yu199195|
+|yuqichou|yuyujulin|yxudong|yymoth|zaunist|
+|zaygrzx|zcai2|zeaposs|zhang98722|zhanghao001|
+|zhangjianweibj|zhangkewei|zhangsean|zhaoyuguang|zhe1926|
+|zhentaoJin|zhousiliang163|zhuCheer|zhyyu|zifeihan|
+|zijin-m|zkscpqm|zoidbergwill|zoumingzm|zouyx|
+|zpf1989|zshit|zxbu|zygfengyuwuzu|undefined|
```

Comment:
undefined? Is it a real GitHub id or script bug?

Reply:
They are old, we are fixing this by using mail prefix. Back to 6 years ago, we don't know need to set squash and merge.